GOVERNANCE.md

```diff
@@ -0,0 +1,70 @@
+# Governance
+
+The governance model adopted in Fluid is influenced by many CNCF projects.
+
+## Principles
+
+- **Open**: Fluid is open source community. 
+- **Welcoming and respectful**: See [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+- **Transparent and accessible**: Work and collaboration should be done in public.
+- **Merit**: Ideas and contributions are accepted according to their technical merit
+  and alignment with project objectives, scope and design principles.
+
+## Project Maintainers
+
+* Classify GitHub issues and perform pull request reviews for other maintainers and the community.
+
+* During GitHub issue classification, apply all applicable [labels](https://github.com/fluid-cloudnative/fluid/labels)
+  to each new issue. Use your best judgement to apply labels, since they are extremely useful for follow-up of future issues. 
```

Comment:
Try your best to label issues which are very helpful and set a good example for future issues.